index.html

```diff
@@ -131,12 +131,12 @@
           task</a></dfn>
         </li>
         <li>
-          <code><dfn><a href=
-          'https://html.spec.whatwg.org/multipage/browsers.html#window'>Window</a></dfn></code>
+          <dfn><a href=
```

Comment:
We need to update these (I mislead you a bit, here is ok to use code, then ReSpec automatically "codes" any `a` that xref this)... we can do these separately tho unless you feel like doing them now. 

```HTML
<code><dfn data-cite="!HTML/multipage/browsers.html#window">Window</dfn></code>
```

See: https://github.com/w3c/respec/wiki/data--cite